eth/tracers/api.go

```diff
@@ -9,6 +11,7 @@ import (
 type TraceConfig struct {
 	*logger.LogConfig
 	Tracer         *string                `json:"tracer"`
+	TracerConfig   *json.RawMessage       `json:"-"`
```

Comment:
```suggestion
	TracerConfig   *json.RawMessage       `json:"tracerConfig"`
```

tracerConfig must be also relayed. Refer to https://github.com/ledgerwatch/erigon/pull/6795 and https://github.com/ledgerwatch/erigon/issues/6758.